sys-fs/e2fsprogs/files/e2fsprogs-1.42.13-protect-existing-fs.patch

```diff
@@ -0,0 +1,90 @@
+diff --git misc/mke2fs.8.in misc/mke2fs.8.in
+index 1002eae..0e7562a 100644
+--- misc/mke2fs.8.in
++++ misc/mke2fs.8.in
+@@ -67,6 +67,9 @@ mke2fs \- create an ext2/ext3/ext4 filesystem
+ [^]\fIfeature\fR[,...]
+ ]
+ [
++.B \-p
++]
++[
+ .B \-q
+ ]
+ [
+@@ -608,6 +611,14 @@ For more information about the features which can be set, please see
+ the manual page
+ .BR ext4 (5).
+ .TP
++.B \-p
++Protect existing partitions, even if run in a script. Prevents
++.B mke2fs
++from overwriting existing partitions, even if running in a script. If a
```

Comment:
I don't think "even if run in a script" needs to be repeated twice. Also the deciding factor is if there is a TTY, not a script right?